impl/src/main/java/org/ehcache/config/builders/ExpiryPolicyBuilder.java

```diff
@@ -64,6 +65,11 @@
     return new TimeToIdleExpiryPolicy(timeToIdle);
   }
 
+  @FunctionalInterface
+  public interface TriFunction<K, V1, V2, R> {
```

Comment:
Could these types be `<T, U, V, R>` to fit with the JDK convention?

Reply:
Yes. I don't know where I took these from. Weird.